src/main/java/com/mraof/minestuck/block/MSBlockShapes.java

```diff
@@ -38,7 +38,19 @@
 	public static final CustomVoxelShape BLENDER = BLENDER_BASE.merge(BLENDER_CONTAINER, BLENDER_HANDLE, BLENDER_LID);
 	
 	public static final CustomVoxelShape CHESSBOARD = new CustomVoxelShape(new double[]{0, 0, 0, 16, 1, 16});
-	
+
+	public static final CustomVoxelShape NAKAGATOR_STATUE = new CustomVoxelShape(new double[]{-1, 0, -1, 17, 2, 17}, new double[]{4, 2, 4, 12, 24, 12});
```

Comment:
Looks like this voxel shape extends outside the 16x16x16 area of the block. Is this fine?

Reply:
It was intentional, it looks like the Capital(head or top piece) of a column and goes on top of brown dungeon columns in the pedestal room

Reply:
Yes, but does it work fine in-game? I know from fences/walls that the collision boxes can extend the 16x16 block without causing problems, but is it really the same with the voxel shape for other uses? I'm not really sure of that. Also, shouldn't the voxel shape reflect the block model? Does that mean that the block model exceeds one block too?